source/how-tos/enable-intel-tsn.rst

```diff
@@ -0,0 +1,311 @@
+.. _tsn:
+
+TSN
+~~~~~~~~~~~~~~~~~
+
+Overview
+--------------------
+
+TSN (Time Sensitive Networking) solution is a set of capabilities that offers Time Synchronization, Traffic Shaping, set of IEEE open standard communication by bringing industrial-grade robustness and reliability to Ethernet . TSN removes the need for physical separation of critical and non-critical communication networks, thus allowing open data exchange between operations and enterprise - a concept at the heart of the Industrial Internel of Things (IIoT).
+To learn more about TSN, see |TSN| for details.
+
+.. |TSN| raw:: html
+
+   <a href="https://www.intel.com/content/www/us/en/industrial-automation/programmable/applications/automation/tsn.html" target="_blank">TSN</a>
+
+
+.. _tsn_container:
+
+TSN container
+--------------------
+
+Container support is a generic SBL feature. Please refer to |Container| and |Container Security| for generic details.
+
+.. |Container| raw:: html
+
+   <a href="https://slimbootloader.github.io/tools/GenContainer.html" target="_blank">Container</a>
+
+.. |Container Security| raw:: html
+
+   <a href="https://slimbootloader.github.io/security/container-security.html?highlight=container#container-security" target="_blank">Container Security</a>
+
+
+SBL defines a container called IPFW for settings related to TSN and TSN is managed as a component inside this IPFW container.
+SBL loads this container and processes it as required including passing the relevant settings to Intel® Firmware Support Package (Intel® FSP) during Runtime.
+SBL defines this container in each platform's build config file BoardConfig.py.
+
+**TSN MAC Address**
+
+User needs to provide TSN MAC address that is unique per board and that can be independently updateable. FBU (Firmware and BIOS Utility) tool is used to create a raw TSN MAC binary.
+This raw TSN binary needs to be signed using the component key that is used during SBL build so that it could pass SBL verification during boot time.
+The signed TSN component is kept into SBL container called IPFW and encapsulate it to create a capsule image. Capsule image is then signed by a firmware update key.

```

Comment:
Remove lines 41 to 48. Same details are presented below in **2. Update using firmware update flow** (line 95)